app/javascript/assets/icons/logout.svg

```diff
@@ -1,3 +1,10 @@
+<<<<<<< HEAD
```

Comment:
Looks like there's a conflict with this one

Reply:
How on earth did the pre commit checks miss that 😕